3rdparty/CMakeLists.txt

```diff
@@ -542,6 +541,11 @@ if (NBL_COMPILE_WITH_SDL2 )
 	    SDL2-static
 	)
 endif()
+if(ANDROID AND NOT ${CMAKE_HOST_SYSTEM_NAME} STREQUAL "Windows")
+	list(APPEND NBL_3RDPARTY_TARGETS
+		OpenSSL
+	)
+endif()
```

Comment:
TODO: @AnastaZIuk ASAP, also make OpenSSL do a fully-out-of-source build without soiling the submodule